ios/Sources/TimetableFeature/TimetableDetailScreen.swift

```diff
@@ -128,19 +142,22 @@ private extension TimetableDetailScreen {
         .padding(.bottom, 84)
     }
 
-    func fixedFooter(tapSlide: @escaping () -> Void, tapVideo: @escaping () -> Void) -> some View {
+    func fixedFooter(timetable: AnyTimetableItem, tapSlide: @escaping () -> Void, tapVideo: @escaping () -> Void) -> some View {
         VStack {
             Spacer()
 
             HStack {
-                VStack {
-                    AssetImage.iconClip.image
-                        .foregroundColor(AssetColor.primary.color)
-                        .frame(width: Const.footerIconSize, height: Const.footerIconSize)
+                if timetable.asset.slideURLString != nil {
+                    VStack {
+                        AssetImage.iconClip.image
+                            .foregroundColor(AssetColor.primary.color)
+                            .frame(width: Const.footerIconSize, height: Const.footerIconSize)
+                    }
```

Comment:
Is this needed to wrap with `VStack` ?
I think below is also okey.

```suggestion
                        AssetImage.iconClip.image
                            .foregroundColor(AssetColor.primary.color)
                            .frame(width: Const.footerIconSize, height: Const.footerIconSize)
```